backend/benefit/applications/services/applications_power_bi_csv_report.py

```diff
@@ -112,3 +111,9 @@ def CSV_COLUMNS(self):
         ]
 
         return columns
+
+    def get_row_items(self):
```

Comment:
Isn't this left unused?

Reply:
No, it overrides the parent ApplicationsCsvService class method so that only one row per application is returned.